--- /tmp/nuphy_stock/keyboards/nuphy/air75_v2/ansi/side_driver.c	2026-06-26 10:29:25.559990448 +0200
+++ /tmp/jincao_qmk/keyboards/nuphy/air75_v2/ansi/side_driver.c	2026-06-26 10:28:10.305265482 +0200
@@ -73,15 +73,15 @@
         // using something like wait_ns(is_one ? T1L : T0L) here throws off timings
         if (is_one) {
             // 1
-            writePinHigh(DRIVER_SIDE_PIN);
+            gpio_write_pin_high(DRIVER_SIDE_PIN);
             wait_ns(WS2812_T1H);
-            writePinLow(DRIVER_SIDE_PIN);
+            gpio_write_pin_low(DRIVER_SIDE_PIN);
             wait_ns(WS2812_T1L);
         } else {
             // 0
-            writePinHigh(DRIVER_SIDE_PIN);
+            gpio_write_pin_high(DRIVER_SIDE_PIN);
             wait_ns(WS2812_T0H);
-            writePinLow(DRIVER_SIDE_PIN);
+            gpio_write_pin_low(DRIVER_SIDE_PIN);
             wait_ns(WS2812_T0L);
         }
     }
